src/Framework/TestCase.php

```diff
@@ -375,10 +375,9 @@ public static function onConsecutiveCalls(...$args): ConsecutiveCallsStub
     }
 
     /**
-     * @param string $name
      * @param string $dataName
      */
-    public function __construct($name = null, array $data = [], $dataName = '')
+    public function __construct(?string $name = null, array $data = [], $dataName = '')
```

Comment:
This is only one small change towards improving the declaration. More PRs for this method will come.